onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -1060,6 +1080,78 @@ ONNX_MS_OPERATOR_SET_SCHEMA(GridSample, 1,
                                   updateOutputShape(ctx, 0, {N, C, H_out, W_out});
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    UnfoldTensor, 1,
+    OpSchema()
+        .SetDoc("Returns a tensor which contains all slices of size size from input tensor in the dimension dim. "
+                "Step between two slices is given by step. "
+                "If sizedim is the size of dimension dim for input tensor, the size of dimension dim in "
+                "the returned tensor will be (sizedim - size) / step + 1. "
+                "An additional dimension of size size is appended in the returned tensor.")
+        .Attr("dim", "specify the dimension to unfold", AttributeProto::INT, static_cast<int64_t>(-1))
+        .Attr("size", "specify the size", AttributeProto::INT)
+        .Attr("step", "specify the step.", AttributeProto::INT, static_cast<int64_t>(1))
+        .Input(0, "input", "input tensor", "T")
+        .Output(0, "output", "Output tensor.", "T")
```

Comment:
Can you specify the expected shapes of the input and output tensors in the description?

Reply:
The description in fact give the information on dims of input and output already.